app-text/master-pdf-editor/master-pdf-editor-5.4.38-r1.ebuild

```diff
@@ -14,26 +14,26 @@ SLOT="0"
 KEYWORDS="~amd64"
 RESTRICT="mirror"
 
-RDEPEND="
-	>=dev-qt/qtcore-5.9.8:5
-	>=dev-qt/qtgui-5.9.8:5
-	>=dev-qt/qtnetwork-5.9.8:5
-	>=dev-qt/qtprintsupport-5.9.8:5
-	>=dev-qt/qtsvg-5.9.8:5
-	>=media-gfx/sane-backends-1.0
-"
+RDEPEND=">=media-gfx/sane-backends-1.0"
 
 QA_PREBUILT="/opt/${PN}/masterpdfeditor5"
```

Comment:
to silent QA warnings:
```
QA_PREBUILT="/opt/${PN}/masterpdfeditor5
        /opt/${PN}/lib/*.so*
        /opt/${PN}/iconengines/*.so*
        /opt/${PN}/platformthemes/*.so*
        /opt/${PN}/printsupport/*.so*
        /opt/${PN}/platforms/*.so*
        /opt/${PN}/imageformats/*.so*
"
```

Reply:
I was wondering about that, thanks.

The required changes were added.